packages/block-serialization-default-parser/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@wordpress/block-serialization-default-parser",
+  "version": "1.0.0",
```

Comment:
I would put `1.0.0-rc.0` or something like that to allow Lerna to do its job - it always bumps version so it would try to do `1.0.1` release otherwise ...